ImputationPipeline/AggregatePRSResults.wdl

```diff
@@ -225,7 +239,7 @@ task BuildHTMLReport {
   >>>
 
   runtime {
-    docker: "us.gcr.io/broad-dsde-methods/tidyverse_kableextra_docker@sha256:e00dd57a1c446e9c07429929893a8399c4a299f0b4fd182245bf45e77c6ab8bd"
+    docker: "us.gcr.io/broad-dsde-methods/mgatzen/tidyverse_kableextra_docker@sha256:fd21f5608a3d43add02f8a8490e49db67f078cb2b906f8cd959a9767350b8c24"
```

Comment:
This needs to be changed to the official tidyverse docker, once it has been pushed

Reply:
You can push the docker image to `us.gcr.io/broad-dsde-methods/tidyverse_kableextra_docker` and update this accordingly